release-configs/qubes-os-iso-full-online.conf

```diff
@@ -0,0 +1,21 @@
+# vim: ft=make
+
+VERBOSE ?= 2
+
+BACKEND_VMM ?= xen
+
+GIT_BASEURL ?= https://github.com
+GIT_PREFIX ?= QubesOS/qubes-
+
+DIST_DOM0 ?=
+DISTS_VM ?=
```

Comment:
This should be filled. Or have a comment what to put here (maybe pointing at other example config).

Reply:
I can fill with default R4.0 value for RELEASE and DIST_DOM0.

Reply:
+1